### PR TITLE
[4.0] packager drone step

### PR DESCRIPTION
**Pull request by zero-24**
https://github.com/joomla/joomla-cms/issues/29013#issuecomment-626304387

As requested by @HLeithner

### CODE REVIEW

**Comment by HLeithner** [comment]
Thank you very much.

**Comment by richard67** [comment]
Ha I just wanted to submitt my positive test result ;-)

File: .drone.yml [hunk 1]
```diff
@@ -354,10 +328,16 @@ steps:
     commands:
       - if [ $DRONE_REPO_NAME != 'joomla-cms' ]; then echo "The packager only runs on the joomla/joomla-cms repo"; exit 0; fi
       - /bin/drone_build.sh
+    volumes:
+      - name: reference
+        path: /reference
+    when:
```
Comment (wilsonge) [inline .drone.yml:334]: Is there any reason for this to be on branch 4.0-dev? given the packager step is running on all branches now?

Comment (HLeithner) [reply .drone.yml:334]: It would also run on security which is not what we want but I think forbid it would be better.

**Pull request merged**